resources/assets/components/pages/AccountPage/Rewards/RewardLevelsTable.js

```diff
@@ -25,6 +25,28 @@ export const userLevelLabel = badgeNumber => {
 
   return userLevel;
 };
+
+const tableSubTitleCopy = badgeCount => {
+  let subTitleCopy =
+    'After you earn your first 2 badges, you’ll start earning rewards!';
+
+  if (userLevelLabel(badgeCount) === 'Legend') {
+    subTitleCopy = `You currently enjoy all the perks of a ${userLevelLabel(
+      badgeCount,
+    )} (you’ll earn 4 chances to win a campaign’s scholarship instead of 1)!`;
+  }
+  if (userLevelLabel(badgeCount) === 'SuperDoer') {
+    subTitleCopy = `You currently enjoy all the perks of a ${userLevelLabel(
+      badgeCount,
+    )} (you’ll earn 3 chances to win a campaign’s scholarship instead of 1)!`;
+  }
+  if (userLevelLabel(badgeCount) === 'Doer') {
+    subTitleCopy = `You currently enjoy all the perks of a ${userLevelLabel(
+      badgeCount,
+    )} (you’ll earn 2 chances to win a campaign’s scholarship instead of 1)!`;
+  }
+  return subTitleCopy;
+};
```

Comment:
Hmmm maybe we can arrange these in an object, so we can simply return the property value per badge value?

e.g. something like

```suggestion
const tableSubTitleCopy = {
  default: 'After you earn your first 2 badges, you’ll start earning rewards!',
  Legend: `You currently enjoy all the perks of a ${userLevelLabel(badgeCount)} (you’ll earn 4 chances to win a campaign’s scholarship instead of 1)!`,
  SuperDoer: `You currently enjoy all the perks of a ${userLevelLabel(badgeCount)} (you’ll earn 3 chances to win a campaign’s scholarship instead of 1)!`, 
  Doer: `You currently enjoy all the perks of a ${userLevelLabel(badgeCount)} (you’ll earn 2 chances to win a campaign’s scholarship instead of 1)!`,
}
```

And then below we can return:
```js
tableSubTitleCopy[totalBadges] || tableSubTitleCopy['default']
```